csrc/quantization/cutlass_w8a8/moe/get_group_starts.cuh

```diff
@@ -48,6 +49,20 @@ __global__ void get_group_gemm_starts(
             a_tensors.size(1), per_act_token, per_out_ch);                 \
   }
 
+#define __CALL_GET_STARTS_KERNEL_FP16(ABC_TENSOR_TYPE, ABC_TYPE)              \
```

Comment:
nit: It's confusing for the reader to have `FP16` in the name since this handles both fp16 and bf16.

I think we should either rename it to something like `__CALL_GET_STARTS_KERNEL_16BIT` or call it `__CALL_GET_STARTS_KERNEL` and append `fp8` to the `float_e4m3_t` version

Reply:
I'm now explicitly writing whether it's 8_bit or 16_bit code wherever applicable